examples/client/src/SetupClient.cpp

```diff
@@ -12,6 +12,9 @@
 // file.
 
 #include "SetupClient.hpp"
+#if !defined(PATH_MAX)
+#define PATH_MAX 4096
```

Comment:
Where did that come from?

Reply:
It's used by `SetupClient` but is not defined. There's no a really portable `PATH_MAX` definition.
Good news is that it's used in tests and utils only.